bundle-size/github-utils.js

```diff
@@ -51,12 +51,7 @@ class GitHubUtils {
   constructor(github, log, cache) {
     this.github = github;
     this.log = log;
-    this.cache =
-      cache ||
-      new NodeCache({
-        'stdTTL': CACHE_APPROVERS_TTL_SECONDS,
-        'checkperiod': CACHE_CHECK_SECONDS,
-      });
+    this.cache = cache || new NodeCache({'checkperiod': CACHE_CHECK_SECONDS});
```

Comment:
Intentional removal of `stdTTL`?

Reply:
Yes, it's a relic from the earlier implementation. There is no standard TTL for this cache, as each cached object has its own TTL. Updating the PR description